src/com/sahokia/kos/concerttickets/test/TicketServiceTest.java

```diff
@@ -0,0 +1,37 @@
+package com.sahokia.kos.concerttickets.test;
+
+import com.sahokia.kos.concerttickets.model.Admin;
+import com.sahokia.kos.concerttickets.model.Client;
+import com.sahokia.kos.concerttickets.model.Ticket;
+import com.sahokia.kos.concerttickets.model.User;
+import com.sahokia.kos.concerttickets.service.TicketService;
+
+public class TicketServiceTest {
```

Comment:
Please don't use the Test in class names if it doesn't contains java tests (unit/integration ones, which is marked @Test)
Lets rename to application and put it on root level - com.sahokia.kos.concertticket

Reply:
The class is renamed as "TicketServiceApplication" and removed on the root level